modules/tinymce/src/core/test/ts/browser/keyboard/FormatShortcutsTest.ts

```diff
@@ -0,0 +1,113 @@
+
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+import { describe, it } from '@ephox/bedrock-client';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyHooks, TinySelections, TinyAssertions, TinyContentActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.keyboard.FormatShortcutsTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+  }, []);
+
+  const platform = PlatformDetection.detect();
+
+  const testFormat = (editor: Editor, style: string) => {
+    TinyContentActions.keystroke(editor, style.charCodeAt(0), platform.os.isMacOS() ? { meta: true } : { ctrl: true } );
+  };
+
+  const Heading = (editor: Editor, number: string) => {
+    TinyContentActions.keystroke(editor, number.charCodeAt(0), platform.os.isMacOS() ? { ctrl: true, alt: true } : { shift: true, alt: true });
+  };
+
+  it('TINY-2884: should set the selection to be bold', () => {
+    const editor = hook.editor();
+    editor.setContent('<p>abc</p>');
+    TinySelections.setSelection(editor, [ 0, 0 ], 0, [ 0, 0 ], 3);
+    testFormat(editor, 'B');
+    TinyAssertions.assertContent(editor, '<p><strong>abc</strong></p>');
+  });
+  it('TINY-2884: should set the selection to be italic', () => {
```

Comment:
As noted via DM's functions, it blocks, etc... should have a new line between them for readability so please fix this file. I'll leave a suggestion here just to make it clearer what I meant.
```suggestion

  it('TINY-2884: should set the selection to be italic', () => {
```

Reply:
You haven't fixed all the issues here like I had asked and instead have only fixed the single instance I left an example on.